src/frontend/view/utils.ts

```diff
@@ -0,0 +1,64 @@
+import { languageName } from '../../common/utils/lang'
+
+export const formatNames = (array: []) =>
+  [...new Set(array)].map((x) => `“${x}”`).join(', ')
+
+export const formatDateTime = (date: any) =>
```

Comment:
Can you handle null date values? At the moment it will break when date is null. Maybe for now return "n/a" for those.